fe/fe-core/src/main/java/com/starrocks/rpc/BackendServiceProxy.java

```diff
@@ -67,9 +67,14 @@ private BackendServiceProxy() {
         if (Config.enable_brpc_share_channel) {
             rpcOptions.setShareThreadPoolUnderEachProxy(true);
             rpcOptions.setShareChannelPool(true);
-            rpcOptions.setThreadPoolSize(1);
+            rpcOptions.setMaxTotoal(Config.brpc_number_of_concurrent_requests_processed_for_share_channel);
```

Comment:
Do we expect users to change this config? Or, change this config could get a better performance? if not, I think we needn't this config.

Reply:
Maybe this config could give me a chance to change it, when users have problems with too high concurrency. 
If we remove the branch `enable_brpc_share_channel=false`, maybe we could use the original config `brpc_number_of_concurrent_requests_processed`?

Reply:
But have it possible some users already change `brpc_number_of_concurrent_requests_processed` in their config file?